mysql-test/r/percona_set_var_extension.result

```diff
@@ -150,14 +125,6 @@ DROP PROCEDURE p1;
 # The hint at statement should have higher preference to the
 # hint at CALL statement
 ###########################################################
-CREATE PROCEDURE p1() BEGIN
-CALL test_hint("SET_VAR(myisam_sort_buffer_size=600000)", "myisam_sort_buffer_size");
-CALL test_hint("SET_VAR(myisam_repair_threads=9)", "myisam_repair_threads");
-CALL test_hint("SET_VAR(sort_buffer_size=420000)", "sort_buffer_size");
-CALL test_hint("SET_VAR(max_join_size=200200200)", "max_join_size");
-CALL test_hint("SET_VAR(innodb_lock_wait_timeout=299)", "innodb_lock_wait_timeout");
-CALL /*+ SET_VAR(innodb_lock_wait_timeout=499)*/ test_hint("SET_VAR(innodb_lock_wait_timeout=299)", "innodb_lock_wait_timeout");
```

Comment:
where this block now? I dont see them in any result file? :thinking:

Reply:
p1() is still created, but I've modified the *.test file with disable/enable_query_log as this part is not crucial to have in the test output and just makes test output less readable.
https://github.com/percona/percona-server/pull/4639/files#diff-313ebf1405c4654787b94ce95e149015abf1dbf2e4961acc7b525a0dc5d76683R90

Reply:
ah, disabled with query_log